wiki/Tournaments/oMMFF/1/en.md

```diff
@@ -0,0 +1,238 @@
+---
+tags:
+- osu! Multi Mode French Fiesta
+- oMMFF
+- MMFF
+---
+# osu! Multi Mode French Fiesta
+
+![oMMFF Logo](img/logo.png)
+
+The **osu! Multi Mode French Fiesta** (***oMMFF***) is a French double elimination one-on-one tournament hosted by ![][flag_FR] [Kasumii-sama](/users/6177263) for the osu!, osu!taiko, osu!catch, and the osu!mania game modes. It is the 1st installment of the osu! Multi Mode French Fiesta.
+
+## Tournament Schedule
+
+### osu!
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-11-04/2018-12-02 |
+| Live Drawings | 2018-12-02 (17:00 UTC) |
+| Qualifiers | 2018-12-15/2018-12-16 |
+| Round of 32 | 2019-01-05/2019-01-06 |
+| Round of 16 | 2019-01-12/2019-01-13 |
+| Quarterfinals | 2019-01-19/2019-01-20 |
+| Semifinals | 2019-01-26/2019-01-27 |
+| Finals | 2019-02-02/2019-02-03 |
+| Grand Finals | 2019-02-09 |
+
+### osu!taiko
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-11-10/2018-12-09 |
+| Live Drawings | 2018-12-09 (17:00 UTC) |
+| Qualifiers | 2019-01-05/2019-01-06 |
+| Round of 32 | 2019-01-12/2019-01-13 |
+| Round of 16 | 2019-01-19/2019-01-20 |
+| Quarterfinals | 2019-01-26/2019-01-27 |
+| Semifinals | 2019-02-02/2019-02-03 |
+| Finals | 2019-02-09/2019-02-10 |
+| Grand Finals | 2019-02-16 |
+
+### osu!catch
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-11-14/2018-12-16 |
+| Live Drawings | 2018-12-16 (17:00 UTC) |
+| Qualifiers | 2019-01-05/2019-01-06 |
+| Round of 32 | 2019-01-12/2019-01-13 |
+| Round of 16 | 2019-01-19/2019-01-20 |
+| Quarterfinals | 2019-01-26/2019-01-27 |
+| Semifinals | 2019-02-02/2019-02-03 |
+| Finals | 2019-02-09/2019-02-10 |
+| Grand Finals | 2019-02-16 |
+
+### osu!mania
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-11-04/2018-12-23 |
+| Live Drawings | 2018-12-23 (17:00 UTC) |
+| Qualifiers | 2019-01-12/2019-01-13 |
+| Round of 32 | 2019-01-19/2019-01-20 |
+| Round of 16 | 2019-01-26/2019-01-27 |
+| Quarterfinals | 2019-02-02/2019-02-03 |
+| Semifinals | 2019-02-09/2019-02-10 |
+| Finals | 2019-02-16/2019-02-17 |
+| Grand Finals | 2019-02-23 |
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for osu! Multi Mode French Fiesta winners. It is possible that these items change with every installment of the osu! Multi Mode French Fiesta.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge, 4 months of osu!supporter tag, [AUKEY KM-G9](https://www.amazon.fr/gp/product/B071KX6DW6/ref=s9_dcacsd_dcoop_bw_c_x_3_w) keyboard, 10€ PayPal/Amazon balance, physical trophy |
```

Comment:
might wanna change the link to https://www.aukey.com/products/87-key-mechanical-keyboard-km-g9?